pyspades/server.py

```diff
@@ -236,13 +236,16 @@ async def update(self):
                         not player.peer.reliableDataInTransit):
                     player.continue_map_transfer()
             # Update world
-            while (time.monotonic() - self.world_time) > UPDATE_FREQUENCY:
+            while (time.monotonic() - self.world_time) >= UPDATE_FREQUENCY:
                 self.loop_count += 1
                 self.world.update(UPDATE_FREQUENCY)
-                try:
-                    self.on_world_update()
-                except Exception:
-                    traceback.print_exc()
+                if (time.monotonic() - start_time) < 2:
```

Comment:
I'd want this to be a constant, something like `UPDATE_DEFICIT_MAX`. It would also be good to only get the value of `time.monotonic()` once so that is does not change during the update. That can cause weird behavior.

Reply:
> It would also be good to only get the value of time.monotonic() once so that is does not change during the update.

But it should change during the update. This is the point.